src/react/databrowser/objects/MetadataSearch.jsx

```diff
@@ -58,11 +58,16 @@ const MetadataSearch = ({ isMetadataType, errorZenkoMsg }: Props) => {
   const suggestionsRef = useRef(null);
   const inputRef = useRef<SearchMetadataInput<> | null>(null);
   useOutsideClick(suggestionsRef, () => setHintsShown(false));
+  const searchInput = query.get('metadatasearch');
 
   const handleChange = e => {
     setInputText(e.target.value);
   };
 
+  useEffect(() => {
+    setInputText(searchInput);
+  }, []);
```

Comment:
Instead of an effect I think this could be done when initiating input text state.
```js
const query = useQuery();
const searchInput = query.get('metadatasearch');
const [inputText, setInputText] = useState(searchInput || '');
```
That would avoid one useless rerender.